app/models/match_prioritization/days_homeless_veterans_families_youth.rb

```diff
@@ -20,6 +20,15 @@ def self.prioritization_for_clients(scope, match_route:) # rubocop:disable Lint/
       scope.order(vets, family, is_youth, age_youth, days)
     end
 
+    def self.supporting_column_names
+      [
+        :veteran,
+        :family_member,
+        :is_currently_youth,
```

Comment:
Same here right?
```suggestion
        :calculated_is_currently_youth,
```

Reply:
yes! thanks for catching these. I've added them to the PR